### DISCUSSION TITLE
Stability and smooth operation

### DISCUSSION THREAD

**Discussion by Frew777** [xe5700/kvmd-armbian#9] [General]
Hi everyone, I wanted to know how smoothly and stably the system works. In 1080p 30 or 60 fps.
Are there only two requirements for a TV box? Availability an otg function and armbian?

**Comment by peacokswiss** [comment]
Hey the TV Box i don't now,   in USB Capture Device some Like-PI  (rock, orangepi etc..pp)  have less interrupts for USB and speed.
So by higher Resolution you will get trouble , like hanging.... ok ;)

yes i run Armbian on orangepizero2 ,  usb capture, it works..   best you let the standard resoultion in kvmd,  and use only LAN cables.
